activemq/activemq.xml

```diff
@@ -0,0 +1,151 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<!-- START SNIPPET: example -->
+<beans
+		xmlns="http://www.springframework.org/schema/beans"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+  http://activemq.apache.org/schema/core http://activemq.apache.org/schema/core/activemq-core.xsd">
+	<!-- Allows us to use system properties as variables in this configuration file -->
+	<bean class="org.springframework.beans.factory.config.PropertyPlaceholderConfigurer">
+		<property name="locations">
+			<value>file:${activemq.conf}/credentials.properties</value>
+		</property>
+	</bean>
+
+	<!-- Allows accessing the server log -->
+	<bean id="logQuery" class="io.fabric8.insight.log.log4j.Log4jLogQuery"
+		  lazy-init="false" scope="singleton"
+		  init-method="start" destroy-method="stop">
+	</bean>
+
+	<!--
+		The <broker> element is used to configure the ActiveMQ broker.
+	-->
+	<broker xmlns="http://activemq.apache.org/schema/core" brokerName="localhost" dataDirectory="${activemq.data}"
+			schedulerSupport="true">
+
+		<destinationPolicy>
+			<policyMap>
+				<policyEntries>
+					<policyEntry topic=">">
+						<!-- The constantPendingMessageLimitStrategy is used to prevent
+							 slow topic consumers to block producers and affect other consumers
+							 by limiting the number of messages that are retained
+							 For more information, see:
+
+							 http://activemq.apache.org/slow-consumer-handling.html
+
+						-->
+						<pendingMessageLimitStrategy>
+							<constantPendingMessageLimitStrategy limit="1000"/>
+						</pendingMessageLimitStrategy>
+					</policyEntry>
+					<policyEntry queue=">">
```

Comment:
This is saying that for every queue (the ">" character) it will use a Dead Letter Queue using queue name prefixed with `DLQ.`